src/typegraph/python/typegraph/utils.py

```
@@ -89,3 +90,30 @@ def freeze_tg_output(
     )
+
+
+@dataclass
+class ResolvedModule:
+    module: str
+    deps: List[str]
+    func_name: str
+
+
+class Module:
+    def __init__(self, source: str, deps: List[str] = []):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)